### PR TITLE
Revise & fix lint issues in trace_mgr.js

### PR DESCRIPTION
**Pull request by mrkmarron**
- [x] `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes
- [x] documentation is changed or added

##### Affected core subsystem(s)

core

##### Description of change
Lint and updates.

PR-URL: https://github.com/mrkmarron/node-chakracore/commit/8b7788698929bbfc1636248bb9a0d5d8daef3c96

### CODE REVIEW

File: lib/trace_mgr.js [hunk 1]
```diff
@@ -19,315 +14,330 @@ var fs = undefined;
...
 
 function createTargetDirectory(pth) {
-    //see if it just exists and, if so, just return true
-    try {
-        fs.accessSync(pth, fs.constants.R_OK | fs.constants.W_OK | fs.constants.X_OK);
-        return true;
+  //see if it just exists and, if so, just return true
+  var accessok = fs.constants.R_OK | fs.constants.W_OK | fs.constants.X_OK;
+  try {
+    fs.accessSync(pth, accessok);
```
Comment (digitalinfinity) [inline lib/trace_mgr.js:191]: Is this now redundant with the new statSync call?

**Review by digitalinfinity — APPROVED** [review]
LGTM if CI is green (@kunalspathak)

**Comment by kunalspathak** [comment]
I don't think CI is necessary because of jslint only changes and we don't have test cases for ttd.

**Comment by digitalinfinity** [comment]
linting doesn't run in CI?

**Comment by kunalspathak** [comment]
It does, but it doesn't fail the job if there are failures. That's the reason my earlier jobs passed despite these jslint issues. As long as mark confirmed that all jslint errors are gone, we should be fine.

**Comment by digitalinfinity** [comment]
Ah I see- good to know

**Comment by kunalspathak** [comment]
Landed in c0297990bab22acd01ed844af011c1c5d4249ff3

**Pull request closed without merging**